--- minicom.orig/src/config.c	2009-06-06 23:19:36.000000000 +0200
+++ minicomsrc/config.c	2009-11-21 12:37:17.151703178 +0100
@@ -51,14 +51,10 @@
   char *p;
 
   /* Read global parameters */
-  if ((fp = fopen(parfile, "r")) == NULL) {
-    fprintf(stderr,
-            _("minicom: there is no global configuration file %s\n"), parfile);
-    fputs(_("Ask your sysadmin to create one (with minicom -s).\n"), stderr);
-    exit(1);
-  }
+  if ((fp = fopen(parfile, "r")) != NULL) {
   readpars(fp, CONFIG_GLOBAL);
   fclose(fp);
+  }
   /* Read personal parameters */
   if ((fp = fopen(pparfile, "r")) != NULL) {
     readpars(fp, CONFIG_PERSONAL);
