diff -r cf5266bda91e -r d358a986f9df src/updown.c
--- a/src/updown.c	Thu Apr 01 00:14:53 2010 +0200
+++ b/src/updown.c	Sun Apr 25 13:32:42 2010 +0200
@@ -448,7 +448,6 @@
 void lockfile_create(void)
 {
   int fd, n;
-  char buf[81];
 
   if (!lockfile[0])
     return;
@@ -458,7 +457,13 @@
   if ((fd = open(lockfile, O_WRONLY | O_CREAT | O_EXCL, 0666)) < 0) {
     werror(_("Cannot create lockfile!"));
   } else {
-    snprintf(buf, sizeof(buf),  "%05d minicom %.20s\n", (int)getpid(), username);
+    // previously used lockfile format:
+    // char buf[81];
+    //snprintf(buf, sizeof(buf),  "%05d minicom %.20s\n", (int)getpid(), username);
+    // FHS format:
+    char buf[12];
+    snprintf(buf, sizeof(buf),  "%10d\n", getpid());
+    buf[sizeof(buf) - 1] = 0;
     write(fd, buf, strlen(buf));
     close(fd);
   }
